_layouts/default.html

```diff
@@ -39,6 +39,11 @@
     url: https://github.com/jupyter/
   - title: Mastodon
     url: https://hachyderm.io/@ProjectJupyter
+legal:
+  - title: Legal
```

Comment:
This looks strange, I think it should be `Legal>Privacy`:

<img width="85" alt="image" src="https://github.com/jupyter/jupyter.github.io/assets/2096628/232a97b2-c745-4c0f-ae0a-074ded05523e">